src/toil_scripts/tools/preprocessing.py

```diff
@@ -233,33 +388,44 @@ def run_base_recalibration(job, cores, indel_bam, indel_bai, ref, ref_dict, fai,
     :param str fai: Reference index FileStoreID
     :param str dbsnp: DBSNP VCF FileStoreID
     :param str mem: Memory value to be passed to children. Needed for CI tests
+    :param str bqsr: BQSR recalibration table FileStoreID
     :param bool unsafe: If True, runs gatk UNSAFE mode: "-U ALLOW_SEQ_DICT_INCOMPATIBILITY"
     :return: FileStoreID for the processed bam
     :rtype: str
     """
     work_dir = job.fileStore.getLocalTempDir()
-    file_ids = [ref, fai, ref_dict, indel_bam, indel_bai, dbsnp]
-    inputs = ['ref.fasta', 'ref.fasta.fai', 'ref.dict', 'sample.indel.bam', 'sample.indel.bai', 'dbsnp.vcf']
+    file_ids = [ref, fai, ref_dict, indel_bam, indel_bai, dbsnp, mills]
```

Comment:
As an aside, I don't love the pattern of having two lists that we then zip. It seems really easy to mess up the ordering between the two, etc. Can we factor this pattern out to use a dictionary? CC @jvivian for comment.